client.go

```diff
@@ -192,10 +193,16 @@ type Subscription struct {
 }
 
 type Manifest struct {
-	Name            string            `yaml:"name"`
-	Buildpacks      []string          `yaml:"buildpacks"`
-	HealthEndpoints map[string]string `yaml:"healthEndpoints"`
-	Builders        map[string]string `yaml:"builders"`
+	Name            string            `json:"name"`
+	Buildpacks      []string          `json:"buildpacks"`
+	HealthEndpoints map[string]string `json:"healthEndpoints"`
+	Probes          *LifecycleProbes  `json:"lifecycle"`
```

Comment:
How are `Probes` related to `HealthEndpoints`?

Reply:
I'd be happy to consolidate this with #558; I wasn't able to find an overarching design for what manifest / runtime files should look like, so I added this here but wasn't super-happy with it.

Reply:
Let's land #558 and then I'll extend on that.

I'd sort of like to be able to include most of a PodTemplateSpec in the `runtime.yaml` and/or `manifest.yaml` so that runtimes could advise on things like Env vars (e.g. set `_HANDLER` to `lambda_function.lambda_handler` in https://github.com/evankanderson/klr-buildpack/blob/master/buildpack/bin/build#L31